airbyte-cdk/python/airbyte_cdk/entrypoint.py

```diff
@@ -94,6 +94,7 @@ def run(self, parsed_args: argparse.Namespace) -> Iterable[str]:
         source_spec: ConnectorSpecification = self.source.spec(self.logger)
         try:
             with tempfile.TemporaryDirectory() as temp_dir:
+                os.environ["TMPDIR"] = temp_dir  # set this as default temp directory
```

Comment:
can you add a comment so we remember **why** we're setting the default temp directory